src/main/java/com/browserstack/automate/ci/jenkins/BrowserStackReportForBuild.java

```diff
@@ -59,6 +60,8 @@ public BrowserStackReportForBuild(final Run<?, ?> build,
         this.resultAggregation = new HashMap<>();
         this.projectType = projectType;
         this.logger = logger;
+        this.customProxy = customProxy;
+        System.out.println("Initialized BrowserStackReportForBuild with customProxy: " + this.customProxy);
```

Comment:
LOGS present

Reply:
Removed the logs in 82915726cafc29f8bf88dce8fd8bacc6f450cbfc

Reply:
Resolving